src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  let totalSum = 0;
+
+  for (let day = 1; day <= days; day++) {
+    totalSum = totalSum + pricePerDay;
+  }
+
+  if (days >= 7) {
+    return totalSum - 50;
+  }
+
+  if (days >= 3) {
+    return totalSum - 20;
```

Comment:
Logical issue: The current logic applies a $20 discount for rentals of 3 or more days, even if a $50 discount for 7 or more days has already been applied. Consider using an `else if` statement here to ensure only one discount is applied.